pytest_jira.py

```diff
@@ -435,5 +435,16 @@ def pytest_configure(config):
                 resolved_statuses,
                 config.getvalue('jira_run_test_case'),
             )
+            config._jira = jira_plugin
             ok = config.pluginmanager.register(jira_plugin, "jira_plugin")
             assert ok
+
+
+@pytest.fixture
+def jira_issue(request):
+    def wrapper_jira_issue(issue_id):
+        jira_plugin = getattr(request.config, '_jira')
```

Comment:
I think you need to set this '_jira' attribute somewhere, probably in pytest_configure.